printer_mods/ScramblerUSA/SideSkirtPowerSwitch/README.md

```diff
@@ -0,0 +1,36 @@
+<h2>Side Skirt Power Switch</h2>
```

Comment:
While this does work... it'd be best to mark your headings using standard markdown syntax, for example:
`# Side Skirt Power Switch`
`## Parts required`
`## Printing`
etc.

Reply:
Agreed